### DISCUSSION TITLE
How to receive an authentication code from the client and authenticate it?

### DISCUSSION THREAD

**Discussion by k-ms1998** [k-ms1998/class-action-claim#5] [Q&A] [answered]
What would be the best way to receive an  authentication code from the client and authenticate it from the server?
I tried using Redis before and it was successful, but it seemed like it was an overkill. 
What would be the easiest and simplest way?

**Comment by pip-izony** [comment, marked as answer]
I think E-mail is the best way to receive the authentication because we can create the authentication code using functions.
